modules/virt-early-access-releases.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/updating/upgrading-virt.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-early-access-releases_{context}"]
+= Early access releases
+
+You can gain access to builds in development by subscribing to the *candidate* update channel for your version of {VirtProductName}. These releases have not been fully tested by Red Hat and are not supported, but you can use them on non-production clusters to test capabilities and bug fixes being developed for that version.
```

Comment:
I've not experienced the issue, but a RH SSG entry exists: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

Red Hat > Red{nbsp}Hat